### DISCUSSION TITLE
Model Viewer object AR placement issuses

### DISCUSSION THREAD

**Discussion by rianlukog** [google/model-viewer#2526] [Q&A]
While loading the AR, the object does not seem to get placed in the screen area but rather far away to the right.
Any tips on fixing this? attaching my code below

<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Model Viewer</title>
    <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
    <style>
model-viewer{
    width: 400px;
    height: 600px;
    margin: 0 auto;

}
    </style>
</head>
<body>
    <model-viewer src="SOURCE1" camera-controls  auto rotate ar ios-src="SOURCE2"></model-viewer>
</body>
</html>

**Comment by rianlukog** [comment]
The object seems to be getting placed outside the screen

**Reply by rianlukog** [reply]
https://user-images.githubusercontent.com/85927224/123937080-6f9f0d00-d9b3-11eb-8e52-9049cf1bd7b4.mp4

**Reply by rianlukog** [reply]
I hope this video explains the issue

**Reply by elalish** [reply]
This is not a model-viewer problem, but a Quick Look problem. I would guess your object is far from the origin and they are probably pointing the camera at the origin of the coordinate system. Probably fixable with a transform.

**Reply by rianlukog** [reply]
Where can I find the transform/reset to camera view command ?

**Reply by elalish** [reply]
As in, fix the transform in your glTF file with whatever software you use to edit it. It's not a model-viewer thing. You could try https://threejs.org/editor/
